advocacy_docs/partner_docs/FlywayGuide/04-Configuration.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: 'Configuration of Flyway'
+description: 'Walkthrough of Flyway configuration'
+---
+
+### 3.1 Prerequisites
```

Comment:
Remove numbering for the headings. 
3.1
4.1
4.1.1
etc.  If they are not steps we are not using numbering.